1-mandatory/2-quotegenerator/quotes.js

```diff
@@ -1,3 +1,21 @@
+const text = document.getElementById("quote");
+const author = document.getElementById("author");
+
+function newQuote(){
+  const randomobj = Math.floor(Math.random() * quotes.length);
+  const displayquote = quotes[randomobj].quote;
+  const displayauthor = quotes[randomobj].author;
+
+  if (displayauthor == null) {
```

Comment:
Instead of checking if the displayauthor is `null`, it would be better to use the nullish coalescing operator `?? `to assign a default value to displayauthor if it is `null` or `undefined`